libpod/oci.go

```diff
@@ -394,7 +394,8 @@ func (r *OCIRuntime) updateContainerStatus(ctr *Container) error {
 		if err != nil {
 			ctr.state.ExitCode = -1
 			ctr.state.FinishedTime = time.Now()
-			return errors.Wrapf(err, "no exit file for container %s found", ctr.ID())
+			logrus.Errorf("No exit file for container %s found: %v", ctr.ID())
```

Comment:
Do you need to add err to the end of this?  I see two %*'s, but only one variable.

Reply:
oh nice catch!

Reply:
Oops, fixed